waspc/examples/todoApp/src/server/dbSeeds.ts

```diff
@@ -0,0 +1,35 @@
+import { createTask } from './actions.js'
+import type { DbSeedFn } from '@wasp/dbSeed/types.js'
+
+async function createUser (prismaClient, data) {
+  const { password, ...newUser } = await prismaClient.user.create({ data })
+  return newUser
+}
+
+export const devSeedSimple: DbSeedFn = async (prismaClient) => {
```

Comment:
I also wonder here if it would be better to pass `entities` instead of `prismaClient`, since they are used to `entities` in Operations? But, then `entities` would need to contain all the entitites in them!
Also, they still might want to import prisma client at some point if they need some fancy stuff like raw queries. We do have docs for importing it though. So I am not sure hm.

Reply:
Prisma client is probably ok here.

Reply:
Ok thanks that helps! I could have also let them import it, but I prefer this, this way it is clear we want them to use it plus it makes mocking possibly easier.